glean-core/src/database/mod.rs

```diff
@@ -109,6 +115,24 @@ impl Database {
         }
     }
 
+    /// Returns `True` if the storage has the given metric.
```

Comment:
I usually phrase the documentation on things like these slightly different (the true/false is implied and denoted by the return type):

> Determine if the storage has the given metric

Reply:
(then again Rust's own libstd documentation uses the same way as you did here, I would just prefer `true` [lower case], we're not Python 🐍 here :P)

Reply:
I changed it as you suggested in the first place to sidestep the whole issue.